### ISSUE TITLE
/paper reload not reloads the anti-xray config

### ISSUE THREAD

**Issue by officialdakari** [PaperMC/Paper#4762]
**Is your feature request related to a problem? Please describe.**
Yes. When I use /paper reload the anti-xray configuration not reload.

**Describe the solution you'd like**
Please make /paper reload reloads the anti-xray config.

**Describe alternatives you've considered**
No alternatives.

**Additional context**
Version: 1.16.1-R0.1-SNAPSHOT (git-Paper-79)

**Comment by BillyGalbreath** [comment]
Some options only take hold when the world initially loads. This is one of those options. It is impossible to reload antixray options unless you unload and load the world using a world managing plugin, like MultiverseCore. But even then, you still wont be able to unload the main world, as that is a Bukkit restriction that cannot be bypassed

TLDR; Not possible to fix this.

**Comment by electronicboy** [comment]
Unsupported version, nor do we really care to put in the effort to make *everything* reload with the reload command, not worth it, just restart.

**Comment by stonar96** [comment]
What Billy said is essentially true. I'm aware of this and it is intended behaviour, as stated in my [gist](https://gist.github.com/stonar96/ba18568bd91e5afd590e8038d14e245e) at the very beginning. I just want to put my 2 cents in here and explain why it's not possible or easy to do this. Each `World` has a final instance of `ChunkPacketBlockController(AntiXray)`, which is created in the [constructor](https://github.com/PaperMC/Paper/blob/master/Spigot-Server-Patches/0363-Anti-Xray.patch#L1404) of the class `World` (i.e. when the `World` is created). When chunks are loaded, this instance is used to determine which additional blocks are needed for obfuscating the chunk. Those blocks are added to the palettes of the chunk sections. When the chunk is later sent to a player, Anti-Xray assumes that those blocks were added. Now if you would replace the Anti-Xray instance by a new one that uses more blocks, the server would crash because those new blocks haven't been added to the already loaded chunks. That's why this instance can't be simple replaced by another one on /paper reload. I hope that makes it a bit clearer.
